libtransmission/quark.cc

```diff
@@ -283,6 +283,7 @@ auto constexpr MyStatic = std::array<std::string_view, 401>{ ""sv,
                                                              "recent-relocate-dir-4"sv,
                                                              "recheckProgress"sv,
                                                              "remote-session-enabled"sv,
+                                                             "remote-session-https"sv,
```

Comment:
FTBFS, the quark keys must be sorted alphabetically.

Reply:
fixed